src/Tasks/Microsoft.Build.Tasks.csproj

```diff
@@ -984,6 +982,7 @@
     <PackageReference Include="System.Reflection.Metadata" />
     <PackageReference Include="System.Reflection.TypeExtensions" />
     <PackageReference Include="System.Resources.Writer" />
+    <PackageReference Include="System.Security.Principal.Windows" />
```

Comment:
Be careful about this: its [only supported on windows](https://github.com/dotnet/corefx/blob/2fac89f63c644ab81d936d62c91e6003302e718d/src/System.Security.Principal.Windows/src/System.Security.Principal.Windows.csproj#L8)